Src/FluentAssertions/Execution/LoadableTestFramework.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+
+namespace FluentAssertions.Execution;
+
+internal abstract class LoadableTestFramework : ITestFramework
```

Comment:
This loads the assembly into the app domain.

If people are using fluent assertions, they may never call a native assertion library class. So it'd never get loaded into the app domain.

The late bound scans the app domain for the assembly. But if it hasn't been loaded it'll never find it and so it'll never be hooked up to throw TUnit exceptions.

Reply:
See the xUnit current example. It needs to try and load it if it's available. I need to do the same. Since two libraries need the same thing, I refactored it into a common base class

Reply:
Then I would like some explanation on the class itself in the form of XML comments.

Reply:
Added